wagtail/admin/templates/wagtailadmin/shared/headers/slim_header.html

```diff
@@ -79,14 +79,35 @@ <h1 class="w-sr-only">
                         {% endif %}
 
                         {% if filters %}
-                            {% fragment as filters_icon %}{% icon name="sliders" title=_("Show filters") %}{% endfragment %}
-                            {% dialog_toggle classname="w-filter-button" dialog_id="filters-dialog" text=filters_icon %}
-
-                            {% dialog theme="floating" id="filters-dialog" title=_("Filters") dialog_root_selector="[data-search-form]" %}
-                                {% for field in filters.form %}
-                                    {% formattedfield field %}
-                                {% endfor %}
-                            {% enddialog %}
+                            <div class="w-drilldown" data-controller="w-drilldown" data-action="w-swap:success@document->w-drilldown#updateParamsCount">
+                                {% fragment as toggle_suffix %}
+                                    <span class="w-drilldown__count" hidden></span>
```

Comment:
```suggestion
                                    <span class="w-drilldown__count" data-w-drilldown-target="count" hidden></span>
```

As per above comment.